packages/office-ui-fabric-react/src/components/Panel/Panel.tsx

```diff
@@ -118,6 +119,7 @@ export class Panel extends BaseComponent<IPanelProps, IPanelState> {
     let isRTL = getRTL();
     let isOnRightSide = isRTL ? isLeft : !isLeft;
     const headerTextId = id + '-headerText';
+    const customWidthStyles = (type === PanelType.custom) ? { width: customWidth } : {};
```

Comment:
sorry for the late response, was on vacation. 

IMO, custom width behaviour should be manually handled by the dev. There could be edge-cases where we want to cover more than 80% of the screen (e.g. cover the whole screen expect a left nav menu). Having a custom width that isn't actually custom seems a bit confusing.